compiler/ilgen/TypeDictionary.cpp

```diff
@@ -421,6 +431,15 @@ TypeDictionary::GetFieldType(const char *structName, const char *fieldName)
    return theStruct->getFieldType(fieldName);
    }
 
+uint32_t
+TypeDictionary::OffsetOf(const char *structName, const char *fieldName)
+   {
+   TR_HashId structID = 0;
+   _structsByName->locate(structName, structID);
```

Comment:
Instead of returning 0, I will put an assert instead. This is something that needs to be addressed more generally (later on) as part of issue #500.